cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +794,56 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+const char* gdv_fn_concat_ws_utf8(int64_t context, const char* separator,
```

Comment:
This function should take at least 2 const char* as input and concat those. This is taking only one input (data)

Reply:
Should I do functions that take upt to 5 inputs, like the elt function?

Reply:
yes, that's what you should do

Reply:
Ok, I antecipated myself and already did it